src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -59,6 +59,13 @@
          used to build the project/solution.
   -->
 
+  <PropertyGroup>
+    <!-- Conditional compilation constant for VS2019-specific code.
+         We'd prefer to set this the shared SonarLint.VSSpecificAssemblies.props file, but
+         the IDE isn't picking up the setting if we do that. -->
+    <DefineConstants Condition="($(VisualStudioVersion)=='17.0' AND $(BuildingInsideVisualStudio)=='true') OR $(VSTargetVersion)=='2022'">$(DefineConstants);VS2022</DefineConstants>
+  </PropertyGroup>
```

Comment:
This is copied from one of the other projects that uses conditional compilation e.g. [Infrastructure.VS](https://github.com/SonarSource/sonarlint-visualstudio/blob/ae2101c36530f705e198e837b6c6302d8b61d4ae/src/Infrastructure.VS/Infrastructure.VS.csproj#L10).